libs/net/traffic_generator.py

```diff
@@ -118,25 +129,18 @@ def _stop_process(vm: BaseVirtualMachine, cmd: str) -> None:
         LOGGER.warning(str(e))
 
 
-def _is_process_running(  # type: ignore[return]
-    vm: BaseVirtualMachine, cmd: str
-) -> bool:
+def _is_process_running(vm: BaseVirtualMachine, cmd: str) -> bool:
     try:
-        for sample in TimeoutSampler(
-            wait_timeout=60,
-            sleep=5,
-            func=vm.console,
+        vm.console(
```

Comment:
Removing the time sampler is a big change, I am not against it, but maybe you should split this change to another PR, so you can verify and focus on it separately